pilot/pkg/model/validation.go

```diff
@@ -1455,9 +1455,6 @@ func ValidateServiceRole(name, namespace string, msg proto.Message) error {
 		errs = appendErrors(errs, fmt.Errorf("at least 1 rule must be specified"))
 	}
 	for i, rule := range in.Rules {
-		if len(rule.Services) == 0 {
```

Comment:
Just curious, why this condition no longer needed?

Reply:
Since we're gonna use workload selector, we don't need to specify the services.